Library/Homebrew/utils.rb

```diff
@@ -220,6 +220,13 @@ def self.git_last_commit_date
     HOMEBREW_REPOSITORY.cd { `git show -s --format="%cd" --date=short HEAD 2>/dev/null`.chuzzle }
   end
 
+  def self.git_branch
+    return unless Utils.git_available?
+#    HOMEBREW_REPOSITORY.cd { `git status -b --porcelain 2>/dev/null`.lines.first.chuzzle }[3..-1]
+#    HOMEBREW_REPOSITORY.cd { `git status 2>/dev/null`.lines.first.chuzzle }
+    HOMEBREW_REPOSITORY.cd { `git branch 2>/dev/null | grep '*'`.lines.first.chuzzle }[2..-1]
```

Comment:
> that command wouldn't handle detached head.

In detached head, it will return `nil`. Therefore, it will show `(none)` in config. Such behavior makes sense to me.